quantization/nlp/bert/trt/e2e_tensorrt_bert_example.py

```diff
@@ -63,11 +65,48 @@ def get_next(self):
                 input_ids = np.expand_dims(features[0].input_ids, axis=0)
                 input_mask = np.expand_dims(features[0].input_mask, axis=0)
                 segment_ids = np.expand_dims(features[0].segment_ids, axis=0)
-        data = [{"input_ids": input_ids, "input_mask": input_mask, "segment_ids":segment_ids}]
+        data = [{"input_ids": input_ids, "attention_mask": input_mask, "token_type_ids":segment_ids}]
         self.current_index += self.batch_size
         self.enum_data_dicts = iter(data)
         return next(self.enum_data_dicts, None)
 
+def inference(ort_session, features, example):
```

Comment:
Can we make inference and evaluation more structural? for example, reuse BertDataReader to read data and define BertEvaluator for evaluation, just like what we did in other e2e examples.

Reply:
The reason I didn't use BertDataReader for inference and evaluation is that for some SQUAD examples, they might have more than one feature. Our current BertDataReader only considers one feature even though the example has multiple features. If we want to handle all the features for all the examples and at the same time run it in batch, we might need to modify the BertDataReader.

Reply:
It's good to handle all the features. One thing is we need to make sure calibration are using same amount of features as inference/evaluation, otherwise calibration may not be able to fully represent inference data

Reply:
For evaluation, I think we must handle all the features of each example in order to achieve better accuracy. Therefore, current data reader should be modified for calibration and evaluation to handle all the features.

Reply:
Yes. we should not use same data for both calibration and evaluation. As we did in other e2e model, we should split the data set, i.e. 80% vs 20%, for each. But the key is to use same pre-processing and data/feature handling for both.